packages/calcite-components/src/demos/autocomplete.html

```diff
@@ -0,0 +1,603 @@
+              scale="s"
+              placeholder="Find an address"
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+        <div class="child">
+          <form class="locate-form">
+            <calcite-autocomplete
+              scale="m"
+              placeholder="Find an address"
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+        <div class="child">
+          <form class="locate-form">
+            <calcite-autocomplete
+              scale="l"
+              placeholder="Find an address"
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+      </div>
+
+      <div class="parent">
+        <div class="child right-aligned-text">Disabled</div>
+        <div class="child">
+          <form class="locate-form">
+            <calcite-autocomplete
+              scale="s"
+              disabled
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+        <div class="child">
+          <form class="locate-form">
+            <calcite-autocomplete
+              scale="m"
+              disabled
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+        <div class="child">
+          <form class="locate-form">
+            <calcite-autocomplete
+              scale="l"
+              disabled
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+      </div>
+
+      <div class="parent">
+        <div class="child right-aligned-text">Readonly</div>
+        <div class="child">
+          <form class="locate-form">
+            <calcite-autocomplete
+              scale="s"
+              read-only
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+        <div class="child">
+          <form class="locate-form">
+            <calcite-autocomplete
+              scale="m"
+              read-only
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+        <div class="child">
+          <form class="locate-form">
+            <calcite-autocomplete
+              scale="l"
+              read-only
+              class="locator-autocomplete"
+              name="location"
+            ></calcite-autocomplete>
+          </form>
+        </div>
+      </div>
+
+      <div class="parent">
+        <div class="child right-aligned-text">Loading</div>
```

Comment:
@ashetland @SkyeSeitz not sure if we can do anything by release, but these loading states with Input aren't the best. Previous implementations weren't as common, but in an Autocomplete context, I think we'll see this overlap issue crop up a bunch: 
![Screenshot 2024-12-02 at 2 24 07 PM](https://github.com/user-attachments/assets/559ea6f6-5c40-46c1-af72-d41e21d464de)

Reply:
Yep, agreed.

Reply:
Agreed but this isn't related to only this component. I think this is outside of scope for autocomplete.